File: PRAK604/PRAK604-2210817310015-RIDHANISETIADI.c
#include <stdio.h>
#include <string.h>
int main(void){
    char kode[90],pesan[90];
    int x,y;
    scanf("%[^\n]%*c",&kode);
    scanf("%[^\n]%*c",&pesan);
    int Pkode=strlen(kode),Ppesan=strlen(pesan);
    if (Pkode != Ppesan){
        printf("Panjang kalimat berbeda,pesan palsu");
    }
    else{
        int x=0, y=0 ;
        for(int k=0;k<Pkode;k++){
            if(pesan[k]== ' ' && kode[k]== ' '){
                continue ;
            }
            else if(kode[k]!=pesan[k]){
                printf(" # ");
                x++;
            }
            else if(kode[k]==pesan[k]){
                printf(" * ");
                y++;
            }
        }
        printf("\n* = %i",y);
        printf("\n# = %i",x);
        if(x<=y){
            printf("\npesan Asli");
        }
        else{
            printf("\npesan Palsu");
        }
    }
    return 0;
}

File: PRAK604/PRAK604-2210817310015-RIDHANISETIADI.py
x=0;y=0
asli = input();palsu = input()
kode = len(asli);pesan = len(palsu)
if kode != pesan:
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for a in range(kode):
        if((asli)[a] == palsu[a]):
            if(asli[a]== ' '):
                print(" ",end=' ')
            else:
                print("*",end=' ')
                x += 1
        else:
            print("#", end='')
            y += 1
    print("\n* = ",x)
    print("# = ",y)
    if(x >= y):
        print("pesan Asli")
    else:
        print("pesan palsu")
